nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/state/flow/index.ts

```diff
@@ -469,3 +469,76 @@ export interface FlowState {
     error: string | null;
     status: 'pending' | 'loading' | 'error' | 'success';
 }
+
+export interface Revision {
```

Comment:
`Revision` already exists [1]. `lastModifier?` will need to be added to the existing one but it the same back end object.

[1] https://github.com/apache/nifi/blob/main/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/state/shared/index.ts#L250